src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -588,7 +588,96 @@ The next four parameters are optional, and apply to advanced Azure endpoint conf
 - *staticPublicIP:* A boolean flag to determine whether the public IPs of the infrastructure's VMs should be static. The default value is _true_.
 - *additionalProperties:* Additional JVM properties to configure your ProActive node. The default values allow to handle both PNP and PAMR protocols. You can add your own properties but be aware that removing the current values might cause the deployment to fail.
 
-The Azure infrastructure offers a simple interface to easily create ProActive nodes on Azure. If your project requires a dynamic and highly customizable set of Azure VMs, you can opt for an Azure Scale Set Infrastructure.
+The Azure infrastructure offers a simple interface to easily create ProActive nodes on Azure. If your project requires a dynamic and customizable set of Azure VMs, you can opt for an <<_azure_scale_set_infrastructure>>.
+
+==== Azure Scale Set Infrastructure
+
+We provide a highly customizable infrastructure to deploy ProActive nodes using an https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/overview[Azure Scale Set]. This infrastructure will allow you to automatically adjust the number of deployed virtual machines (VM) in response to the current workload, and according to your configured parameters, e.g. min/max number of nodes, number of nodes per VM, or minimal up-time. Auto-scaling will help you increase your processing capacity during peak computation periods, while keeping your expenses low during less intensive or idle periods.
+
+Whether you require to run your workflows on Linux or on Windows, use standard or custom VM images, do some pre-configuration steps, or mount a shared file system, the Azure Scale Set infrastructure can handle it. This guide describes the requirements and the various configuration parameters available to deploy an Azure Scale Set infrastructure.
+
+===== Pre-Requisites
+
+As in our standard <<_azure_infrastructure>>, ProActive relies on an existing Azure user account with a valid subscription in order to deploy a Scale Set. These are the specific requirements:
+
+- *Azure account:* You should have a valid Azure account, linked to an Azure subscription with enough budget and https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits[service quotas] to deploy the Scale Set that you request.
+- *Azure privileges:* If your account does not have full Admin privileges on your subscription, please make sure you have rights to:
+* Create a https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview#resource-groups[resource group] in a region of your choice.
+* Create a Scale Set in that resource group, and create its underlying resources: Public IP Address, Load Balancer, Network Security Group and Virtual Network.
```

Comment:
Please ensure that every network prerequisite is formulated, e.g. outgoing connectivity to ProActive Resource Manager.

Reply:
In this case it's actually not strictly necessary: the boot script and more precisely the configuration file opens the necessary ports on the scale set VMs to connect to ProActive's RM.

Reply:
Ok, maybe it would be nice to get a big picture of it.